recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -124,28 +295,83 @@
+    "         examples: str = examples, \n",
+    "         model: str ='granite-code:8b') -> str:\n",
+    "    user_prompt = f\"\"\"\n",
+    "        {examples}\n",
+    "        Question:\n",
+    "        {prompt} Make sure you only generate code that is {os_name()}-compatible!\n",
+    "        Answer:\"\"\" \n",
+    "    response = ollama.chat(model=model, messages=[\n",
+    "      {\n",
+    "        'role':'system',\n",
+    "        'content': system_prompt  \n",
+    "      },\n",
+    "      {\n",
+    "        'role': 'user',\n",
+    "        'content': user_prompt\n",
+    "      },\n",
+    "    ])\n",
+    "    answer = response['message']['content']\n",
+    "    print(answer)\n",
+    "    return answer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Here's a bash script that prints the first 50 files found under the current working directory that have been modified within the last week, along with their last modification times:\n",
+      "\n",
+      "    find . -type f -mmin -1440 | head -n 50 | xargs ls -lth\n",
+      "    Explanation:\n",
+      "\n",
+      "        The 'find' command is used to search for files in the current directory and its subdirectories that meet certain criteria. In this case, we are looking for files (indicated by '-type f') that have been modified within the last 7 days (-mmin -1440).\n",
+      "        \n",
+      "        The output of the 'find' command is then piped to the 'head' command, which limits the output to the first 50 lines. This ensures that we only see the most recently modified files.\n",
+      "        \n",
+      "        Finally, the 'xargs' command is used to pass each file name to the 'ls -lth' command, which prints a detailed list of the files along with their last modification times in human-readable format.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "\"Here's a bash script that prints the first 50 files found under the current working directory that have been modified within the last week, along with their last modification times:\\n\\n    find . -type f -mmin -1440 | head -n 50 | xargs ls -lth\\n    Explanation:\\n\\n        The 'find' command is used to search for files in the current directory and its subdirectories that meet certain criteria. In this case, we are looking for files (indicated by '-type f') that have been modified within the last 7 days (-mmin -1440).\\n        \\n        The output of the 'find' command is then piped to the 'head' command, which limits the output to the first 50 lines. This ensures that we only see the most recently modified files.\\n        \\n        Finally, the 'xargs' command is used to pass each file name to the 'ls -lth' command, which prints a detailed list of the files along with their last modification times in human-readable format.\""
```

Comment:
This is a very texty output. Can we (should we?) get it down to something smaller?

Reply:
Do you see a lot of output? Maybe it's the difference between running Jupyter Notebook vs. Lab, because for me this runs in the `Text_to_Shell` directory, so there is not a lot of output.